src/actions.ml

```diff
@@ -1298,6 +1300,10 @@ let suggest_ci_minimization_for_pr = function
       RunAutomatically
   | {labels} when List.exists ~f:(String.equal "kind: infrastructure") labels ->
       Silent "this PR is labeled with kind: infrastructure"
+  | {body} when String.is_substring ~substring:"offer-minimizer: off" body ->
+      Silent
+        "the PR body contains an 'offer-minimizer: off' directive, which turns \
+         off minimization suggestions"
```

Comment:
I think the logic should be reverted: only `Suggest` when finding the string `offer-minimizer: on`
It's common for experienced developers to wipe out the template for some trivial PRs and you don't want the spam in that case.

Reply:
Of course, this doesn't prevent still having the thing opt-out with the template being `offer-minimizer: on` by default.